Week-1/cattyboard/styles.css

```diff
@@ -0,0 +1,140 @@
+* {
+    font-family: 'Lato', sans-serif;
+    margin: 0;
+    padding: 0;
+}
+
+nav{
+    min-width: 4%;
+    height: 80vh;
+    background-color: #F0F0F0;
+    float: left;
+}
+img {
+    background: transparent;
+}
+
+header.inline {
+    width: 96%;
+    height: 45vh;
+    background-color: #454545;
+    float: right;
+}
+
+section.inline {
+    height: 35vh;
+    width: 96%;
+    /* background-color: rgb(72, 82, 235); */
+    float: right;
+}
+
+div.clearfix {
+    clear: both ;
+}
+
+footer {
+    width: 100%;
+    height: 20vh;
+    background-color: #43689B;
+    display: flex;
+    color: white;
+    justify-content: space-between;
+    align-items: center;
+}
+
+footer img{
+    width: 15vh;
+    margin: 10px;
+    padding: 5px;
+}
+
+.innerfooterflex {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+}
+
+.innerfooterflex p {
+    padding: 3px;
+}
+
+nav img{
+    display: block;
+    padding: 3vh 0;
+    width: 15px;
+    margin-left: auto;
+    margin-right: auto;
+}
+
+
+
+/* .fleximgs {
+    display: flex;
+    flex-direction: column;
+} */
+
+
+table{
+    margin: 15px;
```

Comment:
https://github.com/pesto-students/sai-karthik-j-sairamin/pull/4/files#r851652817

Reply:
Resolved this. Now it is 16.